README.md

```diff
@@ -75,6 +75,14 @@ with:
       secret_key: xxx
 ```
 
+```yml
+with:
+  ...
+  backend_config: |-
+    local:
```

Comment:
Pretty dangerous, you'll lose all the state. The action should probably be checking for this and refusing.

Reply:
we can remove this example, however I think allowing any backend including local should be fine. We have the import functionality. We could hypothetically run with a local backend and import every time and everything should work as expected (except the whole team ID thing)

Reply:
Right, so it begs an important question, if local is always possible, most users shouldn't bother configuring a backend. There's no real point to configuring a custom local backend, presumably the popular backends for users who choose to have them would be S3/Azure Blob/GCS.

Reply:
> most users shouldn't bother configuring a backend

This assertion implies that we change the default import behavior to true and set a local default backend, so maybe not dangerous?

I think the preferred method should still be a persistent backend. Importing takes a while and it's somewhat reassuring to know that state exists somewhere if we ever did want to look at it. We can add a bit in the docs about using an ephemeral local backend with import, I'd be okay with that.

Reply:
> We can add a bit in the docs about using an ephemeral local backend with import, I'd be okay with that.

Yep, I'd even go for `local: true` as an action input. If that's not set, the action asserts that backend config is set and isn't local. And if it is set, `import` is implicitly always `true`. Passing explicit local backend config is probably only needed internally for testing.